app/reducers/search.js

```diff
@@ -66,7 +66,7 @@ const searchMapping = {
     title: 'title',
     type: 'Side',
     color: '#E8953A',
-    path: '/pages/info/',
+    path: '/pages/generelt/',
```

Comment:
You should update the search serializer for flatpages to include the `category`, and redirect to the right url.

Reply:
@cherrybeans